tests/integration_tests/performance/test_snapshot_perf.py

```diff
@@ -48,57 +53,67 @@ def _test_snapshot_create_latency(context):
     snapshot_type = context.custom['snapshot_type']
     enable_diff_snapshots = snapshot_type == SnapshotType.DIFF
 
-    logger.info("""Measuring snapshot create({}) latency for microvm: \"{}\",
-kernel {}, disk {} """.format(snapshot_type,
-                              context.microvm.name(),
-                              context.kernel.name(),
-                              context.disk.name()))
-
     # Create a rw copy artifact.
     rw_disk = context.disk.copy()
     # Get ssh key from read-only artifact.
     ssh_key = context.disk.ssh_key()
 
-    # Measure a burst of snapshot create calls.
-    for i in range(SAMPLE_COUNT):
-        # Create a fresh microvm from aftifacts.
-        vm = vm_builder.build(kernel=context.kernel,
-                              disks=[rw_disk],
-                              ssh_key=ssh_key,
-                              config=context.microvm,
-                              enable_diff_snapshots=enable_diff_snapshots)
+    # Test snapshot creation for every supported target version.
+    for target_version in FIRECRACKER_VERSIONS:
+        # v0.23 does not support diff snapshots
+        if "v0.23" in target_version and snapshot_type == SnapshotType.DIFF:
```

Comment:
Do we expect any other exceptions of this kind in the future? If so, we should consider a structure for these exceptions for readability. If not then this is fine.

Reply:
v0.23 supports loading diff snapshots, we should not skip this test.